src/styleguide/mock-data/data-object/api-response-messages.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiResponseMessages } from '@/types/api-response';
+
+/**
+ * Creates an API response messages object.
+ */
+export default function createApiResponseMessages(options?: Partial<ApiResponseMessages>): ApiResponseMessages {
+  return {
+    SUCCESS: [],
```

Comment:
suggestion: as mentioned before, I would recommend to use camelCase as the default.